pkg/util/cloudinit/schema/Dockerfile.quicktype

```diff
@@ -1,15 +1,16 @@
 FROM node:20 AS build
 WORKDIR /quicktype
 
-RUN npm install --prefix /quicktype quicktype
+COPY package.json package-lock.json* ./
+RUN npm ci --prefix /quicktype quicktype
 
 FROM gcr.io/distroless/nodejs20-debian12
 COPY --from=build /quicktype /quicktype
 
 WORKDIR /output
 CMD [ \
     "/quicktype/node_modules/quicktype/dist/index.js", \
-    "--src", "/schema.json", \
+    "--src", "/schema-cloud-config-v1.json", \
```

Comment:
My guess is that this has been broken since https://github.com/vmware-tanzu/vm-operator/pull/312

Reply:
https://github.com/vmware-tanzu/vm-operator/pull/312 was the PR that fixed the issue for BV. I verified it.

Reply:
Checked this, and I never added that file to the commit. Doh!